lib/samson/periodical.rb

```diff
@@ -15,21 +15,30 @@ module Periodical
 
     class ExceptionReporter
       def initialize(task_name)
+        puts "DEBUG:: NEW EXCEPTION REPORTER #{task_name}" if $debug_messages
         @task_name = task_name
       end
 
       def update(time, _result, exception)
+        puts "DEBUG:: IN EXCEPTION REPORTER UPDATE #{exception.inspect}" if $debug_messages
```

Comment:
periodical fix should be it's own PR ?

Reply:
Yeah it should and will be, I'm just using this PR to try to debug why those tests are failing. I can't merge this anyway until I can test it out on staging :/